.editorconfig

```diff
@@ -1,6 +1,6 @@
 root = true
 
-[{src,scripts,tests}/**.{ts,json,js}]
+[{packages,src,scripts,tests}/**.{ts,json,js}]
```

Comment:
That missing part was screwing my editor, which was generating a bunch of unrelated changes

Reply:
This is also means that my editor suddenly caught a bunch of stuff that were not formatted under our `.editorconfig` rule. It obscures a bit the PR but worthwhile imho (e.g., we don't have to dismiss most of the changes when committing anytime we save a file...)